src/components/VCSMenu.tsx

```diff
@@ -164,7 +264,7 @@ export class VCSMenu extends React.Component<VCSMenuProps, VCSMenuState> {
 
   updatePlotReady(value: boolean) {
     this.setState({ plotReady: value });
-    this.varMenuRef.setState({ plotReady: value });
+    //this.varMenuRef.setState({ plotReady: value });
```

Comment:
if the code doesnt need to be there, remove it instead of commenting it out. Commenting things (like debug logs) is great for when you're writing the code, but shouldnt be included in the final product.